retroshare-gui/src/gui/MainWindow.cpp

```diff
@@ -726,9 +726,12 @@ void MainWindow::updateStatus()
     float downKb = 0;
     float upKb = 0;
     rsConfig->GetCurrentDataRates(downKb, upKb);
+	uint64_t down = 0;
+	uint64_t up = 0;
+	rsConfig->GetTrafficSum(down, up);
 
     if (ratesstatus)
-        ratesstatus->getRatesStatus(downKb, upKb);
+        ratesstatus->getRatesStatus(downKb, down, upKb, up);
```

Comment:
plz rename down->down_total_Kb and up->up_total_Kb. Otherwise because we have downKb and down it sounds like the two variables hold the same quantity with different units. The new down and up are in Kb as well right? If not, just chop of the _Kb part.